app/components/gh-members-segment-select.js

```diff
@@ -68,26 +69,28 @@ export default class GhMembersSegmentSelect extends Component {
             class: 'segment-status-paid'
         }];
 
-        // fetch all labels w̶i̶t̶h̶ c̶o̶u̶n̶t̶s̶
-        // TODO: add `include: 'count.members` to query once API is fixed
-        const labels = yield this.store.query('label', {limit: 'all'});
```

Comment:
Removing labels from this component will effect non-labs code. It's used for the default email recipients select.

I'd suggest adding an argument to the component that allows for labels/products to be optional, that way when used for default email recipients select it could show labels+products and in the visibility select it could show only products.

Reply:
Ah yes, good catch, updating